airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -24,7 +24,7 @@
 from contextlib import closing, suppress
 from dataclasses import dataclass
 from datetime import datetime, timedelta
-from typing import TYPE_CHECKING, Generator, cast
+from typing import TYPE_CHECKING, Generator, Protocol, cast
```

Comment:
Small problem - until end of June, Protocol needs to be imported as:

```
from airflow.typing_compat import Protocol
```

Reply:
thanks